docker/config/airflow.cfg

```diff
@@ -931,81 +892,56 @@ in_cluster = True
+# to a keepalive probe, TCP retransmits the probe after `tcp_keep_intvl` seconds.
+tcp_keep_intvl = 30
 
-[kubernetes_annotations]
+# When the `enable_tcp_keepalive` option is enabled, if Kubernetes API does not respond
+# to a keepalive probe, TCP retransmits the probe `tcp_keep_cnt number` of times before
+# a connection is considered to be broken.
+tcp_keep_cnt = 6
 
-# The Key-value annotations pairs to be given to worker pods.
-# Should be supplied in the format: key = value
+# Set this to false to skip verifying SSL certificate of Kubernetes python client.
+verify_ssl = True
 
-[kubernetes_environment_variables]
+[smart_sensor]
+# When `use_smart_sensor` is True, Airflow redirects multiple qualified sensor tasks to
+# smart sensor task.
+use_smart_sensor = False
 
-# The scheduler sets the following environment variables into your workers. You may define as
-# many environment variables as needed and the kubernetes launcher will set them in the launched workers.
-# Environment variables in this section are defined as follows
-# ``<environment_variable_key> = <environment_variable_value>``
-#
-# For example if you wanted to set an environment variable with value `prod` and key
-# ``ENVIRONMENT`` you would follow the following format:
-# ENVIRONMENT = prod
-#
-# Additionally you may override worker airflow settings with the ``AIRFLOW__<SECTION>__<KEY>``
-# formatting as supported by airflow normally.
-
-[kubernetes_secrets]
-
-# The scheduler mounts the following secrets into your workers as they are launched by the
-# scheduler. You may define as many secrets as needed and the kubernetes launcher will parse the
-# defined secrets and mount them as secret environment variables in the launched workers.
-# Secrets in this section are defined as follows
-# ``<environment_variable_mount> = <kubernetes_secret_object>=<kubernetes_secret_key>``
-#
-# For example if you wanted to mount a kubernetes secret key named ``postgres_password`` from the
-# kubernetes secret object ``airflow-secret`` as the environment variable ``POSTGRES_PASSWORD`` into
-# your workers you would follow the following format:
-# ``POSTGRES_PASSWORD = airflow-secret=postgres_credentials``
-#
-# Additionally you may override worker airflow settings with the ``AIRFLOW__<SECTION>__<KEY>``
-# formatting as supported by airflow normally.
+# `shard_code_upper_limit` is the upper limit of `shard_code` value. The `shard_code` is generated
+# by `hashcode % shard_code_upper_limit`.
+shard_code_upper_limit = 10000
 
-[kubernetes_labels]
+# The number of running smart sensor processes for each service.
+shards = 5
 
-# The Key-value pairs to be given to worker pods.
-# The worker pods will be given these static labels, as well as some additional dynamic labels
-# to identify the task.
-# Should be supplied in the format: ``key = value``
+# comma separated sensor classes support in smart_sensor.
+sensors_enabled = NamedHivePartitionSensor
```

Comment:
this file shouldnt have any diff. We should be using the .cfg file from 1.10.12 as is so that the customers get the same configuration settings as in MWAA prod 1.10.12 but only use this branch on local runner for migration purposes